runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -114,6 +115,8 @@ public Layout getCurrentLayout() {
                             + "invalidate view", we.getCorrectEpoch());
                 } else if (re instanceof NetworkException) {
                     log.warn("layoutHelper: System seems unavailable", re);
+                } else if (re instanceof ShutdownException) {
```

Comment:
Should we introduce a different exception (i.e. ServerUnavailable?) because this one is shared by the CorfuRuntime shutdown exception (i.e. shouldn't be retried)